IPython/html/nbextensions.py

```diff
@@ -242,6 +242,62 @@ def install_nbextension(path, overwrite=False, symlink=False, user=False, prefix
             src = path
             _maybe_copy(src, full_dest, verbose)
 
+
+def make_cmdclass(setupfile, path, enable=None):
+    """Build nbextension cmdclass dict for the setuptools.setup method.
+
+    Parameters
+    ----------
+    setupfile: str
+        Path to the setup file.
+    path: str
+        Directory relative to the setup file that the nbextension code lives in.
+    enable: [str=None]
+        Extension to "enable".  Enabling an extension causes it to be loaded
+        automatically by the IPython notebook.
+
+    Usage
+    -----
+    setup(
+        name='flightwidgets',
+        ...
+        cmdclass=make_cmdclass(__file__, 'flightwidgets', 'flightwidgets/init'),
+    )
+    """
+
+    from setuptools.command.install import install
+    from setuptools.command.develop import develop
+    from os.path import dirname, abspath, join
+
+    from .services.config import ConfigManager
+
+    def run_nbextension_install(develop):
+        install_nbextension(join(dirname(abspath(setupfile)), path), symlink=develop)
+        if enable is not None:
+            print("Enabling the extension ...")
+            cm = ConfigManager()
+            cm.update('notebook', {"load_extensions": {enable: True}})
```

Comment:
Load shouldn't be part of installation. That's like adding `import package` to PYTHONSTARTUP at install time.